### PR TITLE
Use secure cookie and make redirect url configurable

### PR DESCRIPTION
**Pull request by meln1k**
# Description

# To-Dos

- [x] Add test coverage for new or updated functionality
- [x] Lint and test with `nox`

# Code of Conduct

By submitting this pull request, I agree to follow the [code of conduct](https://some.engineering/code-of-conduct).

### CODE REVIEW

File: fixbackend/auth/redirect_to_spa.py [hunk 1]
```diff
@@ -28,32 +29,24 @@ class RedirectToSPA(Transport):
...
-                localStorage.setItem("fix-jwt", "{token}");
-                window.location.replace("{self.redirect_path}");
-            </script>
-        </head>
-        <body></body>
-    </html>"""
-
-        response = Response(content=payload, status_code=status.HTTP_200_OK, media_type="text/html")
-        response.set_cookie("fix.auth", value=token)
+        response = RedirectResponse(url=self.redirect_url, status_code=status.HTTP_303_SEE_OTHER)
```
Comment (aquamatthias) [inline fixbackend/auth/redirect_to_spa.py:37]: Why 303? Is this expected to handle post requests?

File: fixbackend/config.py [hunk 2]
```diff
@@ -77,6 +79,8 @@ def parse_args(argv: Optional[Sequence[str]] = None) -> Namespace:
     parser.add_argument("--cdn-bucket", default=os.environ.get("FIXUI_CDN_BUCKET", "fix-ui"))
     parser.add_argument("--fixui-sha", default=os.environ.get("FIXUI_SHA", ""))
     parser.add_argument("--static-assets", type=Path, default=os.environ.get("STATIC_ASSETS"))
+    parser.add_argument("--session-ttl", type=int, default=int(os.environ.get("SESSION_TTL", 3600)))
+    parser.add_argument("--oauth-redirect-url", default=os.environ.get("OAUTH_REDIRECT_PATH", "/"))
```
Comment (aquamatthias) [inline fixbackend/config.py:83]: We do this for development purposes, right?
Please add a comment, describing the purpose.

**Review by aquamatthias — COMMENTED** [review]
1 question / 1 small request. Otherwise lgtm. Thanks.

File: fixbackend/auth/redirect_to_spa.py (hunk 1 above)
Comment (meln1k) [reply fixbackend/auth/redirect_to_spa.py:37]: Semantically 303 is more correct here. The requests coming here is GET from oauth callback URL.

File: fixbackend/config.py (hunk 2 above)
Comment (meln1k) [reply fixbackend/config.py:83]: noted

**Review by aquamatthias — APPROVED** [review]

**Pull request merged**